src/grease.cpp

```diff
@@ -6,6 +6,21 @@
 
 namespace MLS_NAMESPACE {
 
+#ifdef DISABLE_GREASE
+
+Capabilities
+grease(Capabilities&& capabilities, [[maybe_unused]] const ExtensionList& extensions)
+{
+  return capabilities;
+}
+
+ExtensionList
+grease(ExtensionList&& extensions) {
+  return extensions;
+}
```

Comment:
I had started by `ifdef-ing` each call to `grease(...)` but that creates the branching in many places. This limits the scope of the changes required at the cost of keeping calls to `grease` around that do nothing.